src/components/Navigation/Navigation.jsx

```diff
@@ -171,9 +171,9 @@ export default class Navigation extends React.Component {
     let { pageUrl = '' } = this.props;
 
     if (section.children) {
-      return section.children.some(child => pageUrl.includes(`${child.url}/`));
+      return section.children.some(child => { return pageUrl == '/' + child.url + '/'; });
 
-    } else return pageUrl.includes(`${section.url}/`);
+    } else return pageUrl.includes('/' + section.url + '/', 0);
```

Comment:
Why make these changes here and above? This code does the same thing as the template strings did except for the addition of the `0` parameter which basically turns it into a `startsWith`. Can you provide an example of why this fix/fixes is needed?

Reply:
It's a matter of choosing which nav is active; I'm working with it again to make sure it works and no more issue.

Reply:
> It's a matter of choosing which nav is active;

I understand that, my question is what was broken about it that made these changes necessary? When I tested on the local server these items seemed to be activating just fine.

Reply:
Try URL: http://127.0.0.1:3000/concepts/plugins/

Reply:
![capsssssssssssssssssture](https://user-images.githubusercontent.com/12815768/29891025-e0406c62-8df3-11e7-8dcf-09738a547caa.PNG)

Reply:
![nav3](https://user-images.githubusercontent.com/12815768/29891055-fb08c4d6-8df3-11e7-89b4-7d3e584c2794.png)

Reply:
but with my fix, it still wrong. I'm doing with it.

Reply:
This is an issue of `str.includes`.
I'm doing with `RegExp` to determine the most accurate.
The next commit.